keyform/views.py

```diff
@@ -37,6 +38,15 @@ def form_valid(self, form):
             return self.form_invalid(form)
 
     def get_form(self, form_class=None):
-        form = CreateForm(**self.get_form_kwargs())
+        form = CreateForm(instance=Request(staff=get_user_model().objects.all()[0]), **self.get_form_kwargs())
         form.request_formset = RequestFormSet(**self.get_form_kwargs())
         return form
+
+    def get_form_kwaargs(self):
```

Comment:
`get_form_kwaargs()` -> `get_form_kwargs()`
Also figure out if this method is even necessary